mxpy-up.py

```diff
@@ -226,130 +233,239 @@ def install_mxpy(exact_version: str, from_branch: str):
+        return
+
     old_export_directive = f'export PATH="{Path("~/elrondsdk").expanduser()}:$PATH"\t# elrond-sdk'
     new_export_directive = f'export PATH="${{HOME}}/multiversx-sdk:$PATH"\t# multiversx-sdk'
 
-    profile_file = get_profile_file()
-    profile_info_content = profile_file.read_text()
+    profile_files = get_profile_files()
+
+    if not profile_files:
+        print(f"""
+###############################################################################
+No shell profile files have been found.
+
+The "mxpy" command shortcut will not be available until you add the the directory "{sdk_path}" to the system PATH.
+###############################################################################
+Do you understand the above?
+""")
+        confirm_continuation(interactive)
+        return
 
-    logger.info(f"Using shell profile: {profile_file}")
+    profile_files_formatted = "\n".join(f" - {file}" for file in profile_files)
+    profile_files_contents = [profile_file.read_text() for profile_file in profile_files]
+    any_old_export_directive = any(old_export_directive in content for content in profile_files_contents)
+    any_new_export_directive = any(new_export_directive in content for content in profile_files_contents)
 
-    if old_export_directive in profile_info_content:
+    if any_old_export_directive:
         # We don't perform the removal automatically (a bit risky)
-        logger.warning(f"Please manually remove the following entry from the shell profile ({profile_file}): {old_export_directive}.")
+        print(f"""
+###############################################################################
+It seems that the old path "~/elrondsdk" is still configured in shell profile.
 
-    if new_export_directive in profile_info_content:
+Please MANUALLY remove it from the shell profile (now or after the installer script ends).
+
+Your shell profile files:
+{profile_files_formatted}
+
+The entry (entries) to remove: 
+    {old_export_directive}
+###############################################################################
+Make sure you understand the above before proceeding further.
+###############################################################################
+""")
+        confirm_continuation(interactive)
+
+    if any_new_export_directive:
         # Note: in some (rare) cases, here we'll have false positives (e.g. if the export directive is commented out).
-        logger.info(f"multiversx-sdk path ({sdk_path}) is already configured in shell profile.")
-        return
+        print(f"""
+###############################################################################
+It seems that the path "~/multiversx-sdk" is already configured in shell profile.
 
-    logger.info(f"Configuring multiversx-sdk path [{sdk_path}] in shell profile...")
-    logger.info(f"[{profile_file}] is being modified...")
+To confirm this fact, check the shell profile (now or after the installer script ends). 
```

Comment:
Maybe remove the word `fact`? Something like `To confirm this, please check ...` sounds a little better in my opinion.

Reply:
Fixed.